src/components/scenes/WalletDetailsScene.tsx

```diff
@@ -292,19 +274,7 @@ function TransactionListComponent(props: Props) {
       return <EmptyLoader />
     }
 
-    const disableAnimation = index >= MAX_LIST_ITEMS_ANIM
-    if (typeof item === 'string') {
-      return (
-        <EdgeAnim disableAnimation={disableAnimation} enter={{ type: 'fadeInDown', distance: 30 * (index + 1) }}>
-          <SectionHeader title={item} />
-        </EdgeAnim>
-      )
-    }
-    return (
-      <EdgeAnim disableAnimation={disableAnimation} enter={{ type: 'fadeInDown', distance: 30 * (index + 1) }}>
-        <TransactionListRow navigation={navigation as NavigationBase} transaction={item} wallet={wallet} />
-      </EdgeAnim>
-    )
+    return null // We're not using the FlatList rendering anymore
```

Comment:
If we're no longer using FlatList, why hasn't it be replaced with a scrollable SceneWrapper?